tests/components/google_photos/test_services.py

```diff
@@ -38,27 +83,16 @@ async def test_upload_service(
         ]
     )
 
-    with (
-        patch(
-            "homeassistant.components.google_photos.services.Path.read_bytes",
-            return_value=b"image bytes",
-        ),
-        patch(
-            "homeassistant.components.google_photos.services.Path.exists",
-            return_value=True,
-        ),
-        patch.object(hass.config, "is_allowed_path", return_value=True),
-    ):
-        response = await hass.services.async_call(
-            DOMAIN,
-            "upload",
-            {
-                "config_entry_id": config_entry.entry_id,
-                "filename": "doorbell_snapshot.jpg",
-            },
-            blocking=True,
-            return_response=True,
-        )
+    response = await hass.services.async_call(
+        DOMAIN,
+        "upload",
+        {
+            "config_entry_id": config_entry.entry_id,
+            "filename": "doorbell_snapshot.jpg",
+        },
```

Comment:
can we use constants here? (for the service name and parameter keys)

Reply:
Updated.